lib/src/pigeons/gesture_listeners.dart

```diff
@@ -4,13 +4,48 @@
 
 part of mapbox_maps_flutter;
 
+/// A structure that defines additional information about map content gesture.
+class MapContentGestureContext {
+  MapContentGestureContext({
+    required this.touchPosition,
+    required this.point,
+  });
+
+  /// The location of gesture in Map view bounds.
+  ScreenCoordinate touchPosition;
+
+  /// Geographical coordinate of the map gesture.
+  Point point;
+
+  Object encode() {
+    return <Object?>[
+      touchPosition.encode(),
+      point.encode(),
+    ];
+  }
+
+  static MapContentGestureContext decode(Object result) {
+    result as List<Object?>;
+    return MapContentGestureContext(
+      touchPosition: ScreenCoordinate.decode(result[0]! as List<Object?>),
```

Comment:
Is there a way we can avoid the forced `!`?

Reply:
This code is Pigeon-generated, it seems that the assumptions here, since the decoding and encoding is both done by Pigeon, they deem to be acceptable to force unwrap it.

Reply:
Seems fair 👍